manifests/params.pp

```diff
@@ -38,14 +40,25 @@
       $mcodirs = [$mco_dir]
 
       $path_separator = ':'
+
+      $user  = 0
+      $group = 0
     }
     'windows' : {
+      $package_name = 'puppet-agent'
+      $service_names = ['puppet', 'mcollective']
+
+      $local_packages_dir = 'C:\\Program Files\\Puppet Labs\\Puppet Enterprise\\packages'
```

Comment:
Another thought is to use `C:\ProgramData\PuppetLabs` instead (I think there's a PE `common_appdata` fact for that). If you write to `C:\Program Files` then those files will prevent puppet from uninstalling cleanly (it will leave behind the new files and any parent directories). It's not a big deal, but not ideal. Also if you are running puppet as "non-root", puppet may not have permission to write to `C:\Program Files`, but it can write to `C:\ProgramData`